benchopt/plotting/plot_objective_curve.py

```diff
@@ -84,42 +76,28 @@ def plot_objective_curve(df, obj_col='objective_value', plotly=False,
 
         q1 = df_.groupby('stop_val')['time'].quantile(.1)
         q9 = df_.groupby('stop_val')['time'].quantile(.9)
-
+        
         fill_between_x(
-            fig, curve['time'], q1, q9, curve[obj_col], color=CMAP(i % CMAP.N),
-            marker=markers[i % len(markers)], label=solver_name, plotly=plotly
+            fig, curve['time'], q1, q9, curve[obj_col], color=colors[i % CMAP.N],
```

Comment:
could you use `get_solver_color` here in order to make all the color dispatching mechanism happen in this function ? Otherwise we have to maintain it here, and in `get_solver_function`

Reply:
agreed

Reply:
also same for markers?

Reply:
In fact, as Amélie pointed out I find it weird to use `get_solver_color` here, as we would just undo the work that is done by `get_solver_color` (which takes colors[i] and maps it to a string). I'd be in favor of leaving the call to `colors`